bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,364 @@
+Running a manual discovery should show up all devices on your local network. 
+
+There seems to be an issue between OH mDNS implementation and Shelly so that initially the binding is not able to catch the thing’s ip address (in this case the event reports 0.0.0.0 as ip address - this will be ignored) or devices don’t show up all the time.
+To fix this you need to run the manual discovery multiple times until you see all your devices.
+Make sure to wake-up battery powered devices (press the button inside the device) so they show up on the network.
+
+## Binding Configuration
+
+The binding has some global configuration options.
+Go to PaperUI:Configuration:Addons:Shelly Binding to edit those.
+
+| Parameter      |Description                                                    |Mandatory|Default                                         |
+|----------------|---------------------------------------------------------------|---------|------------------------------------------------|
+| defaultUserId  |Default userid for http authentication when not set in thing   |    no   |admin                                           |
+| defaultPassword|Default password for http authentication when not set in thing |    no   |adnub                                           |
+
+
+## Thing Configuration
+
+|Parameter         |Description                                                   |Mandatory|Default                                           |
+|------------------|--------------------------------------------------------------|---------|--------------------------------------------------|
+|deviceIp          |IP address of the Shelly device, usually auto-discovered      |    yes  |none                                              |
+|userId            |The userid used for http authentication*                      |    no   |none                                              |
+|password          |Password for http authentication*                             |    no   |none                                              |
+|lowBattery        |Threshold for battery level. Set alert when level is below.   |    no   |20 (=20%), only for battery powered devices       |
+|updateInterval    |Interval for the background status check in seconds.          |    no   |1h for battery powered devices, 60s for all others|
+|eventsButton      |true: register event "trigger when a button is pushed"        |    no   |false                                             |
+|eventsSwitch      |true: register event "trigger of switching the relay output"  |    no   |true                                              |
+|eventsSensorReport|true: register event "posted updated sensor data"             |    no   |true for sensor devices                           |
+|enableCoIoT       |true: Listen for CoIoT/COAP events, OFF: Don't use COAP       |    no   |true for battery devices, false for others        |
```

Comment:
Should be `eventsCoIoT` instead of `enableCoIoT` as far as I can see.

Reply:
fixed, thx